templates/learn.html

```diff
@@ -28,10 +28,23 @@
       </div>
     </div>
   </a>
-  <a class="card" href="https://github.com/bevyengine/awesome-bevy">
+  <a class="card" href="https://github.com/bevyengine/bevy/tree/latest/examples">
```

Comment:
```suggestion
  <a class="card" href="https://github.com/bevyengine/bevy/tree/latest/examples#examples">
```
with a similar motivation as #69